proto/cmp/types/v1alpha/price.proto

```diff
@@ -18,33 +18,66 @@ message PriceDetail {
   // Principle price element
   Price price = 1;
 
-  // gross price in case taxes or commissions are specified
-  float gross = 2;
-
-  // Identification whether the gross price must be respected towards the end-user
+  // Identification whether the price must be respected towards the end-user
   // and the product or service cannot be offered for a lower price.
-  bool binding = 3;
+  bool binding = 2;
 
   // Description of the product or service this price(element) is valid for.
-  string description = 4;
+  string description = 3;
```

Comment:
Is this a free-text description? 
If yes what use does it have in the context of the protocol?
If not to which rules must it adhere to be valid?

Reply:
Description was for "Early Booking Offer", "Stay 7, pay 6 nights" and extras
example:
CurrencyCode="EUR" Code="ALC" Description="Alcoholic Drinks"

enums for tax types and price type similar to OTA

Reply:
Enums implemented, kept description for the time being